extensions/amp-list/0.1/amp-list.js

```diff
@@ -690,29 +739,38 @@ export class AmpList extends AMP.BaseElement {
       return;
     }
     this.measureElement(() => {
-      const scrollHeight = target./*OK*/scrollHeight;
+      let targetHeight = target./*OK*/scrollHeight;
       const height = this.element./*OK*/offsetHeight;
-      if (scrollHeight > height) {
-        this.attemptChangeHeight(scrollHeight).then(() => {
-          if (this.loadMoreEnabled_) {
-            this.mutateElement(() => {
-              this.loadMoreButton_.classList
-                  .remove('i-amphtml-list-load-more-overflow');
-            });
-          }
-        }).catch(() => {
-          if (this.loadMoreEnabled_) {
-            this.mutateElement(() => {
-              this.loadMoreButton_.classList
-                  .add('i-amphtml-list-load-more-overflow');
-              this.element.appendChild(this.loadMoreButton_);
-            });
-          }
-        });
+      if (this.loadMoreEnabled_) {
+        const buttonHeight = this.loadMoreButton_./*OK*/scrollHeight;
```

Comment:
you probably want offsetHeight. ScrollHeight would include overflow area (in case someone puts an overflowing load more button) which won't be correct.